doc/releases/migration-guide-3.5.rst

```diff
@@ -94,6 +94,11 @@ Recommended Changes
   marked as deprecated as well. The new modem subsystem :kconfig:option:`CONFIG_MODEM_CMUX`
   and :kconfig:option:`CONFIG_MODEM_PPP`` should be used instead.
 
+* Device drivers should now be restricted to ``PRE_KERNEL_1``, ``PRE_KERNEL_2``
+  and ``POST_KERNEL`` init levels. Other levels, including ``EARLY``,
```

Comment:
s/Other levels/Device initialization in other levels, including .../

Otherwise it sounds like the levels are being deprecated in general.

or ...

s/have been deprecated and will be removed in future releases./have been deprecated and device initialization will not be poissible using those levels in future releases./

Reply:
added some clarifications, I think there's no ambiguity now.